main.js

```diff
@@ -8,4 +30,4 @@ const constructAll = function () {
 
 document.addEventListener('o.DOMContentLoaded', constructAll);
 
-export {oComments, oCommentsCount};
+export { Comments, CommentCount };
```

Comment:
I'm confused about what I would need to export...essentially, I want an app (e.g. next-article) to pull in comments _or_ comment count by calling `oComments.init()`.

Reply:
However, I can't really do that with the `constructAll` function above, no? I think I may be confusing myself...

Reply:
I think my other comment maybe covers this? With my suggestion, the export becomes the new `CommentsController` class (almost definitely not that name the more I think about it)